vectorstores/alloydb/vectorstore_test.go

```diff
@@ -121,10 +150,8 @@ func TestPingToDB(t *testing.T) {
 
 func TestApplyVectorIndexAndDropIndex(t *testing.T) {
 	t.Parallel()
-	vs, cleanUpTableFn, err := setVectorStore(t)
-	if err != nil {
-		t.Fatal(err)
-	}
+	envVariables := getEnvVariables(t)
+	vs, cleanUpTableFn := vectorStore(t, envVariables)
```

Comment:
Each test needs a different table name if running in parallel. There are race conditions of clean and droping the table

Reply:
or just remove the run in parallel